scripts/xsslint/xsslint/linters.py

```diff
@@ -856,13 +883,30 @@ class MakoTemplateLinter(BaseLinter):
     """
     LINE_COMMENT_DELIM = "##"
 
+
+    ruleset = RuleSet(
+        mako_missing_default='mako-missing-default',
+        mako_multiple_page_tags='mako-multiple-page-tags',
+        mako_unparseable_expression='mako-unparseable-expression',
+        mako_unwanted_html_filter='mako-unwanted-html-filter',
+        mako_invalid_html_filter='mako-invalid-html-filter',
+        mako_invalid_js_filter='mako-invalid-js-filter',
+        mako_js_missing_quotes='mako-js-missing-quotes',
+        mako_js_html_string='mako-js-html-string',
+        mako_html_entities='mako-html-entities',
+        mako_unknown_context='mako-unknown-context',
+        python_wrap_html='python-wrap-html',
+    ) + visitors.ruleset
```

Comment:
Do you need the visitors.ruleset added here if you are getting them with the python.ruleset below?  This also has a bad smell, but may just be a copy/paste issue.

Reply:
Yes, because a custom python_linter can be passed to the MakoLinter that may use a different set of rules.